app/views/api/v1/needs/show.json.jbuilder

```diff
@@ -0,0 +1,11 @@
+json.ignore_nil!
+json.needs [@need] do |need|
+  json.cache! need do
+    json.partial! @need unless @need.nil?
+  end
+end
+
+json.meta do
+  json.success @success
+  json.msg @msg
```

Comment:
APIs with random words abbreviated make the whole API less predictable. You start with things like "msg" for "message" and soon you're asking was it "error" or "err", etc.? Even though we are operating quickly and under stress, I think it's worth investing the time in the extra characters, because this stuff is hard to change later.